apps/health_campaign_field_worker_app/lib/main.dart

```diff
@@ -37,22 +70,45 @@ class MainApplication extends StatelessWidget {
               TableHideActionBloc(const TableHideActionState()),
         ),
       ],
-      child: BlocBuilder<AuthBloc, AuthState>(builder: (context, state) {
-        return MaterialApp.router(
-          theme: DigitTheme.instance.mobileTheme,
-          routeInformationParser: appRouter.defaultRouteParser(),
-          routerDelegate: AutoRouterDelegate.declarative(
-            appRouter,
-            navigatorObservers: () => [AppRouterObserver()],
-            routes: (handler) => [
-              if (state.isAuthenticated)
-                const AuthenticatedRouteWrapper()
-              else
-                const UnauthenticatedRouteWrapper(),
-            ],
-          ),
-        );
-      }),
+      child: BlocBuilder<ApplicationConfigBloc, ApplicationConfigState>(
+        builder: (context, appConfigState) {
+          return BlocBuilder<AuthBloc, AuthState>(
+            builder: (context, authState) {
+              final appConfig =
+                  appConfigState.appConfigDetail?.configuration?.appConfig;
+
+              return MaterialApp.router(
+                supportedLocales: const [
+                  Locale('en', 'IN'),
+                  Locale('hi', 'IN'),
```

Comment:
this is temporarily added for testing? Can we move these to some constant or config so we know this has to be changed later?

Reply:
@egov-joy  Updated